modules/aws/route53/main.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
```

Comment:
should we call this file provider rather than main? Just that way we know what it is before going into it?

Reply:
We should perhaps document our recommended layout, but having main.tf as the default/first place to look in a module follows the conventions: https://developer.hashicorp.com/terraform/tutorials/modules/module-create#module-structure

Reply:
I agree but following this structure would this not suggest that the code in dns.tf and alieses.tf should actually be in main.tf? seen as these are the main part of the module code?

Reply:
everything else looks fine so once we make a decision on this I will approve :)